Create Dev-portal-overview.md (#737)
bigcommerce/dev-docs: docs/api-docs/partner/Dev-portal-overview.md

# Developer Portal Overview

<div class="otp" id="no-index">

### On this page
- [About BigCommerce apps](#about-bigcommerce-apps)
- [Use cases](#use-cases)
- [Developing your app](#develop-your-app)
- [Resources](#resources)
</div> 

> 

Reviewer: Developer Portal is the resource where you can create, manage and publish your application for the BigCommerce App Marketplace.

Author: I added this line

> Using BigCommerce’s powerful APIs, SDKs, and toolkits, developers can register and build new apps on the developer portal that are installable on BigCommerce stores. Once approved, you can sell apps (or made available free of charge) to all BigCommerce merchants on the [Apps Marketplace](https://www.bigcommerce.com/apps/). You will need a [DevTools](https://devtools.bigcommerce.com/my/apps) account to register the draft app and a BigCommerce store to test the installation.

Reviewer: ```suggestion
Once your application is approved, you can either sell it to BigCommerce merchants or make it installable for free. 

To get started with developing BigCommerce applications, you will need to create a [Developer Portal](https://devtools.bigcommerce.com/my/apps) account to register and submit your application for review. You will also need to start a [free store trial](https://www.bigcommerce.com/essentials/free-trial/) to test your application against. 
```

> ## About BigCommerce apps

Reviewer: This section appears to be a copy of the information provided in Introduction to Building Apps. Instead of including sections of that article, I think we should link to it instead.
```suggestion
## About BigCommerce apps
```

### Create Custom Widgets
Create [widgets](https://support.bigcommerce.com/s/article/Page-Builder#builder) for merchants to install and configure.

[Learn more about creating widgets](https://developer.bigcommerce.com/api-docs/storefront/widgets/widgets-overview).

## Developing your app

Here's how to develop your app with BigCommerce:

1. [Start a free trial](https://www.bigcommerce.com/essentials/free-trial/) to test your apps.

2. [Apply to be a Partner](https://www.bigcommerce.com/partners) to get paid for your apps.

3. [Create a DevTools account](https://devtools.bigcommerce.com/) to register and submit apps.

4. [Join the Developer Community](https://support.bigcommerce.com/s/group/0F913000000HLjECAW/bigcommerce-developers) to get help from other BigCommerce developers.
5. [Check out the Marketplace](https://www.bigcommerce.com/marketplace/) to get inspiration on what to build.

6. [Decide which type of app to build](https://developer.bigcommerce.com/api-docs/apps/guide/types).

7. [Start building apps with CodeSandbox, Node, and Express](https://developer.bigcommerce.com/api-docs/apps/quick-start).

## Resources

> ### Sample Apps

Reviewer: This information is provided in the Resources article. 
```suggestion

```

* [Python / Flask](https://github.com/bigcommerce/hello-world-app-python-flask)
* [PHP / Silex](https://github.com/bigcommerce/hello-world-app-php-silex)
* [Ruby / Sinatra](https://github.com/bigcommerce/hello-world-app-ruby-sinatra)
* [Laravel / React](https://github.com/bigcommerce/laravel-react-sample-app)
* [Node / FaunaDB / Nelify](https://github.com/bigcommerce/channels-app/)

### Tools

> * [Node API Client](https://github.com/getconversio/node-bigcommerce)

Reviewer: ```suggestion

```

> * [Python API Client](https://github.com/bigcommerce/bigcommerce-api-python)

Reviewer: ```suggestion

```

> * [PHP API Client](https://github.com/bigcommerce/bigcommerce-api-php)

Reviewer: ```suggestion

```

> * [Ruby API Client](https://github.com/bigcommerce/bigcommerce-api-ruby)

Reviewer: ```suggestion

```

> * [Ruby OmniAuth Gem](https://github.com/bigcommerce/omniauth-bigcommerce)

Reviewer: ```suggestion

```

> * [Big Design Developer Playground](https://developer.bigcommerce.com/big-design/)

Reviewer: ```suggestion
* [BigDesign Developer Playground](https://developer.bigcommerce.com/big-design/)
```

* [Figma UI Kit](https://www.figma.com/file/jTVuUkiZ1j3rux8WHG4IKK/BigDesign-UI-Kit?node-id=0%3A1/duplicate)
* [Adobe Illustrator UI Kit](https://design.bigcommerce.com/bigdesign-ui-kit)

> ### Blog Posts

Reviewer: ```suggestion
### Blog Posts
- [Introduction to Building Apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro)
```

> * [How to Test App Authentication Locally with ngrok](https://medium.com/bigcommerce-developer-blog/how-to-test-app-authentication-locally-with-ngrok-149150bfe4cf) (BigCommerce Developer Blog)

Reviewer: This blog post should be added to the Authentication article. 
```suggestion

```

* [Building a BigCommerce App Using Laravel and React](https://medium.com/bigcommerce-developer-blog/building-a-bigcommerce-app-using-laravel-and-react-711ceceb5006) (BigCommerce Developer Blog)

> * [Big Design Tutorial](https://medium.com/bigcommerce-developer-blog/bigdesign-build-native-looking-uis-with-the-bigcommerce-design-system-fb06a01a24f2) (BigCommerce Developer Blog)

Reviewer: ```suggestion
* [BigDesign Tutorial](https://medium.com/bigcommerce-developer-blog/bigdesign-build-native-looking-uis-with-the-bigcommerce-design-system-fb06a01a24f2) (BigCommerce Developer Blog)
```

# Developer Portal Overview

<div class="otp" id="no-index">

### On this page
- [About BigCommerce apps](#about-bigcommerce-apps)
- [Resources](#resources)
</div> 

> The Developer Portal is the resource developers use to create, manage, and publish their apps for the BigCommerce App Marketplace (https://www.bigcommerce.com/apps/). Once your app is approved, you can either sell it to BigCommerce merchants or make it installable for free. 

Reviewer: ```suggestion
The Developer Portal is the resource developers use to create, manage, and publish their apps for the BigCommerce [App Marketplace](https://www.bigcommerce.com/apps/). Once your app is approved, you can either sell it to BigCommerce merchants or make it installable for free. 
```

To get started with developing BigCommerce apps, you will first need to create a [Developer Portal](https://devtools.bigcommerce.com/my/apps) account. You will use your Developer Portal account to register and submit your app for review. You will also need to start a [free store trial](https://www.bigcommerce.com/essentials/free-trial/) to test your app. 

For information on app development, see [Introduction to Building Apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro).

For instructions on how to use the Developer Portal to create an app, see Managing Apps in DevTools.

## Resources

* [Python / Flask](https://github.com/bigcommerce/hello-world-app-python-flask)
* [PHP / Silex](https://github.com/bigcommerce/hello-world-app-php-silex)
* [Ruby / Sinatra](https://github.com/bigcommerce/hello-world-app-ruby-sinatra)
* [Laravel / React](https://github.com/bigcommerce/laravel-react-sample-app)

> * [Node / FaunaDB / Nelify](https://github.com/bigcommerce/channels-app/)

Reviewer: ```suggestion
* [Node / FaunaDB / Netlify](https://github.com/bigcommerce/channels-app/)
```

List products on external [sales channels](https://www.bigcommerce.com/apps/sales-channels/) with [Channels Toolkit](https://developer.bigcommerce.com/api-docs/channels/channels-toolkit-reference).

[Learn more about connecting apps to sales channels.](https://developer.bigcommerce.com/api-docs/channels/overview).

### Add and Manage Scripts
Create and manage scripts with [Scripts API](https://developer.bigcommerce.com/api-reference/store-management/scripts) and [Script Manager](https://support.bigcommerce.com/s/article/Using-Script-Manager).

[Learn more about creating scripts](https://developer.bigcommerce.com/api-docs/storefront/scripts-overview).

### Create Custom Widgets
Create [widgets](https://support.bigcommerce.com/s/article/Page-Builder#builder) for merchants to install and configure.

[Learn more about creating widgets](https://developer.bigcommerce.com/api-docs/storefront/widgets/widgets-overview).

## Developing your app

Here's how to develop your app with BigCommerce:

1. [Start a free trial](https://www.bigcommerce.com/essentials/free-trial/) to test your apps.

2. [Apply to be a Partner](https://www.bigcommerce.com/partners) to get paid for your apps.

> 

Reviewer 2: @traci707 Can you remove the new lines between each step? It causes extra spacing.

Author: removed lines
